### PR TITLE
Merge branch 'fix/basename-4' into 'development'

### PR DESCRIPTION
**Pull request by ee-ehab-xx**
Merge branch 'fix/basename-4' into 'development'

### CODE REVIEW

**Comment by zoli18153** [comment]
minden jót mindenkinek

**Comment by Traihoha** [comment]
Ok

**Comment by Macfred11** [comment]
Good job

**Comment by Fitrah81** [comment]
Test

**Comment by congho1804** [comment]
> Merge branch 'fix/basename-4' into 'development'

**Comment by anilk666** [comment]
> Merge branch 'fix/basename-4' into 'development'

Ok

File: src/components/Account.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const NameValueTable = ({data, decodeValue = false}) => {
 }
 
 const balanceRow = bal => (
-  <tr key={bal.asset_code ? `${bal.asset_code}-${bal.asset_issuer}` : 'XLM'}>
+  <tr key={bal.asset_code ? `${bal.asset_code}-${bal.asset_issuer}` : 'test-π'}>
     <td>
       <Asset
         type={bal.asset_type}
```
Comment (anilk666) [inline src/components/Account.js:70]: I will give you 100

**Review by anilk666 — CHANGES_REQUESTED** [review]

**Review by Isaibrahim71998 — COMMENTED** [review]
Is mostly be an wallet or account no to pay

**Comment by Slyman042** [comment]
> Merge branch 'fix/basename-4' into 'development'

Superb

**Comment by Slyman042** [comment]
Waooo

**Comment by Olcay222** [comment]
GABD5PVZZO44UOXXPRTG3Q5KPCHXBDKLFA6VD4UDD5NUMJYVG54M7ZWS

**Comment by Olcay222** [comment]
@wwolcayto@gmail.com

**Comment by Olcay222** [comment]
Olcay 222

**Comment by Jessicadrichardson** [comment]
Ok

**Review by Anne0506 — APPROVED** [review]

**Comment by Hariommeena123** [comment]
Kab tak Dal na h pesa Lena h ke nih aaap ko

**Review by Kittykat72766 — APPROVED** [review]

**Review by Kittykat72766 — APPROVED** [review]

**Review by Ahmedjamal66 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Comment by Robin8004** [comment]
> Is mostly be an wallet or account no to pay

Tnx

**Review by Robin8004 — APPROVED** [review]

**Review by DevinAPieceOfSit — COMMENTED** [review]
@noxious

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]

**Review by Robin8004 — APPROVED** [review]
Ok

**Review by DevinAPieceOfSit — APPROVED** [review]
pi-apps:development FR:@noxious15

**Review by christiannolia — APPROVED** [review]

**Comment by 077258200** [comment]
[C704FF9BA82C511B4A4C4767C66A610C71CB37F2511217CC5A3C59DFF2FE906DJSON](2$) 

> 

> ``

**Comment by Rickm6045** [comment]
Hello

**Review by FukYaMudda — APPROVED** [review]

**Comment by RRAAJJUUDDAASS** [comment]
RRAAJUUDDAASS

**Comment by Asijoon** [comment]
Hello good time 
in the browser Do you include the Persian language translator?
There was more or less in the Pii Ke program, Search and Escape
Please post here too. The number of Iranian users is not small
Thank you

**Review by Thienan007 — COMMENTED** [review]
Screenshot_2022-07-16-21-17-28-89

**Review by Thienan007 — COMMENTED** [review]
Screenshot_2022-07-16-21-17-28-89

**Review by janbic1822 — APPROVED** [review]

**Comment by Thienan007** [comment]
Ok

**Comment by Ilyasmalik7788** [comment]
Yes

**Comment by Ilyasmalik7788** [comment]
Yes

**Comment by Thienan007** [comment]
Ok

**Review by Thienan007 — COMMENTED** [review]
Ok

**Comment by Asahan63** [comment]
Go...

**Review by ananddeshpande1976 — COMMENTED** [review]
How to contribute in this project

**Comment by SHADYKIMBUM** [comment]
> Merge branch 'fix/basename-4' into 'development'

**Comment by Emanuelcarvalho2004** [comment]
Nice

**Comment by Emanuelcarvalho2004** [comment]
Nice 

> Hello good time in the browser Do you include the Persian language translator? There was more or less in the Pii Ke program, Search and Escape Please post here too. The number of Iranian users is not small Thank you

**Comment by Emanuelcarvalho2004** [comment]
Nice

**Comment by Emanuelcarvalho2004** [comment]
Nice 

> Hello good time in the browser Do you include the Persian language translator? There was more or less in the Pii Ke program, Search and Escape Please post here too. The number of Iranian users is not small Thank you

**Comment by Emanuelcarvalho2004** [comment]
Nice

**Comment by wawanar** [comment]
kyc

**Review by Mosesesmart — CHANGES_REQUESTED** [review]
Thanks 🙏

**Review by solbage — APPROVED** [review]

**Review by Thienan007 — APPROVED** [review]
Ok

**Review by Thienan007 — APPROVED** [review]
Ok

**Comment by Theindra87** [comment]
Let't go..

Pada tanggal Sab, 13 Agt 2022 21.11, Asahan63 ***@***.***>
menulis:

**Comment by nutty323** [comment]
Merge

**Comment by nutty323** [comment]
Merge branch fix

**Comment by nutty323** [comment]
Merge

**Comment by nutty323** [comment]
Merge branche/fix basename

**Comment by nutty323** [comment]
Megre/fix

**Comment by Itzjboy** [comment]
So friends what do you say about this platform

**Comment by Mamatbat** [comment]
Hi

**Comment by mrJambil1** [comment]
Good job 👍

**Comment by ZatmanOptionz24** [comment]
##í

**Comment by Costas1948** [comment]
Gpod

**Comment by minepicomosasrer00** [comment]
Thank you very much

**Review by Fitra126 — APPROVED** [review]
pi-apps:development

**Review by Fitra126 — APPROVED** [review]

**Review by nanis94 — COMMENTED** [review]
Dockerfile

**Review by 09167372994 — APPROVED** [review]

**Review by 09167372994 — APPROVED** [review]

**Comment by Capojazzy1** [comment]
Kyc no resolved till now why?

**Comment by Capojazzy1** [comment]
Approve all changes to real pi .

**Comment by Capojazzy1** [comment]
#331

**Comment by maripaungmai** [comment]
{
  "_links": {
    "self": {
      "href": "https://api.testnet.minepi.com/ledgers/13125163"
    },
    "transactions": {
      "href": "https://api.testnet.minepi.com/ledgers/13125163/transactions{?cursor,limit,order}",
      "templated": true
    },
    "operations": {
      "href": "https://api.testnet.minepi.com/ledgers/13125163/operations{?cursor,limit,order}",
      "templated": true
    },
    "payments": {
      "href": "https://api.testnet.minepi.com/ledgers/13125163/payments{?cursor,limit,order}",
      "templated": true
    },
    "effects": {
      "href": "https://api.testnet.minepi.com/ledgers/13125163/effects{?cursor,limit,order}",
      "templated": true
    }
  },
  "id": "5dfdf63cd4477f63ffd8d1a6c76d891529515d852fd85b14dfa9a5978f98963e",
  "paging_token": "56372145839669248",
  "hash": "5dfdf63cd4477f63ffd8d1a6c76d891529515d852fd85b14dfa9a5978f98963e",
  "prev_hash": "a9636bfa20d0d0d1d1366585091f1cd2036d2aacd6ffdadd2317ca9ec88452f8",
  "sequence": 13125163,
  "successful_transaction_count": 15,
  "failed_transaction_count": 0,
  "operation_count": 15,
  "tx_set_operation_count": 15,
  "closed_at": "2023-07-21T21:19:28Z",
  "total_coins": "100000000000.0000000",
  "fee_pool": "2065756.6818620",
  "base_fee_in_stroops": 100000,
  "base_reserve_in_stroops": 100000000,
  "max_tx_set_size": 1000,
  "protocol_version": 15,
  "header_xdr": "AAAAD6lja/og0NDR0TZlhQkfHNIDbSqs1v/a3SMXyp7IhFL4Sly87VDaZODYILjJ3w4kbfzOIpyELXJjPOFkaIsTIY0AAAAAZLr2YAAAAAAAAAABAAAAAL0dIh9YCWKVtdZZd7HLMYRYlv5sCibPT1h2qqXnmjFeAAAAQBIwIrQ8ND/B9Ru2f66HYLNo/+TGd0GYSa/6bLB4pfh7jV1ktY8pbv3YQ0KKQkn//GC+mfzLkJiqnEKj7mFRfQVDpzCUkeYT7S7Q1NSMcXWlRv4CKvLoCw0Baozi/LFEK2ozXvMB5ZHrawoCwPVds8R+Btj6NV+USTpeMo/OKedgAMhGKw3gtrOnZAAAAAASybbuoTwAAAAAAAAAAAAEd0oAAYagBfXhAAAAA+g+qr+XlTRdNy9lrbxaE0+af0wwSStiyLBQLgHvgN6XtTs9isnuB91PdyBf57KJLEc11YfpdbDLj5Pq8FRWrHE+ra6IhywbYyBOzjkUXZA9SuO1oWb+ShTuaWwwULYDTw7jT60k40S6PT0u8/h4O3mmpzB9KtG7ZUOKo1vSLzNKngAAAAA="
}

**Comment by Camquyen1996** [comment]
https://sandbox.minepi.com/app/ng-c-linh-804cd7895c93abc4

**Comment by Steveai25** [comment]
Meeting with the reviewers

**Comment by Darknes990** [comment]
help me get pi

**Comment by Dauda080** [comment]
At what price are you buying it

**Review by Dezzj — COMMENTED** [review]
Reviewed

**Review by Dezzj — COMMENTED** [review]
Veiwed

**Comment by chatch** [comment]
not related to stellar explorer - only the pi fork ...

**Comment by Asijoon** [comment]
Hello, I was one of the pioneers of Pie, for some reason, the program was
deleted from my phone
Now I can't log in to my account
I corresponded several times, I tried very hard, but...
Please, if possible, help me log into my account










Your success is my wish

در تاریخ جمعه ۲ سپتامبر ۲۰۲۲،‏ ۰:۲۳ Emanuelcarvalho2004 <
***@***.***> نوشت:

**Comment by 301166** [comment]
Pi name Sauvos

**Comment by 301166** [comment]
Pi day is happy day!

**Comment by 301166** [comment]
All pi day is happy day!🙂

**Comment by Tysomi** [comment]
Great day

File: src/components/Account.js
Comment (Sutherlandjoseph) [inline src/components/Account.js:1]: Nive

**Comment by TESLA-SATI** [comment]
LOGO
SATI ; )

**Pull request closed without merging**